metadata.rb

```diff
@@ -3,7 +3,7 @@
 maintainer_email 'foss@gilles.cornu.name'
 license 'Apache 2.0'
 description 'Installs Google Android SDK'
-version '0.2.0'
+version '0.2.1'
```

Comment:
Also need to bump the version number in [`metadata.rb`](https://github.com/gildegoma/chef-android-sdk/blob/master/metadata.rb#L6)